frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -214,22 +246,25 @@ export class Wizard {
     } else if (template) {
       await this.selectTemplate(template);
     } else {
-      if (provisionSource) {
-        await this.selectProvisionSource(provisionSource);
-      } else {
-        throw Error('VM Provision source not defined');
-      }
       if (os) {
         await this.selectOperatingSystem(os);
       } else {
         throw Error('VM OS not defined');
       }
+
+      if (provisionSource && !_.isEqual(provisionSource, ProvisionSource.DISK)) {
```

Comment:
can you use a simpler logic please?

```tsx
      if (provisionSource) {
        if (provisionSource === ProvisionSource.DISK) {
          await this.disableGoldenImageCloneCheckbox();
        }
        await this.selectProvisionSource(provisionSource);
      } else {
        throw Error('VM Provision source not defined');
      }
```

Reply:
but we said `provisionSource === ProvisionSource.DISK` doesn't work

Reply:
it works if you clone it properly - please see https://github.com/openshift/console/pull/6850#discussion_r504755763